conversations/v3/common.yaml

```diff
@@ -473,24 +483,24 @@ components:
         mediaId:
           type: string
           format: uuid
+          description: The media ID provided by tyntec
         contentType:
           type: string
+          description: The media type
         validity:
           type: string
           format: date-time
+          description: For how long is the media stored in minutes
         fileName:
           type: string
+          description: The name of the file
         fileSize:
           type: integer
+          description: The size of the file in bytes
     Problem:
       type: object
       additionalProperties: true
       description: The problem object follows the RFC-7807 (https://tools.ietf.org/html/rfc7807)
-      required:
-        - type
-        - title
-        - status
-        - detail
```

Comment:
Several examples use only the `status` property. On the other hand, once, I got another problem object without the `status` property. The RFC seems to make all properties optional too. Is that right?

Reply:
that's true.

at which occasion have you received an information without the status?

as we try to send all 4 information always.

Reply:
For example:
```shell
$ # the response to a missing latitude does not include detail
$ curl -X POST https://api.tyntec.com/conversations/v3/messages -H "apikey: $APIKEY" -H 'Content-Type: application/json' -H 'accept: application/json' --data '{"from": "4923147790719", "to": "420605723965", "channel": "whatsapp", "content": {"contentType": "location", "location": {"longitude": 7.4954884, "latitude": "a"}}}'
{"type":"https://docs.tyntec.com/problems","title":"Data given was not parseable","status":400}
$
$ # the response to an unsupported Content-Type does not include type
$ curl -X POST https://api.tyntec.com/conversations/v3/messages -H "apikey: $APIKEY" -H 'Content-Type: application/foo' -H 'accept: application/json' --data '{"from": "4923147790719", "to": "420605723965", "channel": "whatsapp", "content": {"contentType": "location", "location": {"longitude": 7.4954884, "latitude": 51.5005765}}}'
{"title":"Unsupported Media Type","status":415,"detail":"415 UNSUPPORTED_MEDIA_TYPE \"Content type 'application/foo' not supported\""}
$
$ # the response to a missing body does not include type and detail
$ curl -X POST https://api.tyntec.com/conversations/v3/messages -H "apikey: $APIKEY" -H 'Content-Type: application/json' -H 'accept: application/json' --data ''
{"title":"Message request missing","status":400}
$
$ # the response to a missing API key is completely different
$ curl -X POST https://api.tyntec.com/conversations/v3/messages -H 'Content-Type: application/json' -H 'accept: application/json'
{"message":"No API key found in request"}
```
I am not sure whether I did not receive `status` in any other case.

Reply:
thank you :) got it, 

so only from the very external system the status code is not provided